supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -174,13 +174,13 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 [discrete]
 [[bios]]
 ==== image:images/yes.png[yes] BIOS (noun)
-*Description*: "BIOS" is an abbreviation for basic input and output system. The plural form is "BIOSes".
+*Description*: "BIOS" is an abbreviation for basic input and output system. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware in older PCs. Modern computers have a different kind of firmware, called either EFI or UEFI. If you need to refer to computer firmware, do not use BIOS as a general term. Use _firmware_ or a specific phrase like "UEFI firmware or legacy BIOS" instead.
```

Comment:
```suggestion
*Description*: "BIOS" is an abbreviation for Basic Input/Output System. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware that provides runtime services for operating systems in older PCs. Modern computers use a different kind of firmware, called either EFI or UEFI. 

Do not use _BIOS_ as a generic term to refer to this type of firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.
```
- Should be "Basic Input/Output System". See https://en.wikipedia.org/wiki/BIOS.
- A few other suggestions to tighten things up a bit.